src/renderer/helpers/api/invidious.js

```diff
@@ -312,12 +313,18 @@ export function filterInvidiousFormats(formats, allowAv1 = false) {
   // Which is caused by Invidious API limitation on AV1 formats (see related issues)
   // Commented code to be restored after Invidious issue fixed
   //
-  // if (allowAv1 && av1Formats.length > 0) {
-  //   return [...audioFormats, ...av1Formats]
-  // } else {
-  //   return [...audioFormats, ...h264Formats]
-  // }
-  return [...audioFormats, ...h264Formats]
+  // As we generate our own DASH manifest (using YouTube.js) for multiple audio track support in Electron,
+  // we can allow AV1 in that situation. If we aren't in electron,
+  // we still can't use them until Invidious fixes the issue on their side
+  if (process.env.IS_ELECTRON) {
+    if (allowAv1 && av1Formats.length > 0) {
+      return [...audioFormats, ...av1Formats]
+    } else {
```

Comment:
Suggestion: Remove this else block (including return)

Reply:
I always wonder why early return pattern is not used more often